ledger/complete/wal/checkpointer.go

```diff
@@ -656,8 +656,6 @@ func readCheckpoint(f *os.File, dir string, filename string) ([]*trie.MTrie, err
 		return readCheckpointV4(f)
 	case VersionV5:
 		return readCheckpointV5(f)
-	case VersionV6:
-		return ReadCheckpointV6(dir, filename)
```

Comment:
> If we keep this, then we can't merge this into master, but only in feature branch.

Maybe I'm missing something, but isn't this PR targeting another PR which targets the feature branch?

> I'd like to merge the feature branch to master, even though the ReadCheckpointV6 and WriteCheckpointV6 are not being used yet.
> And I could have a separate branch to enable it, so that that PR will have minimum changes waiting to be merged when we consider the feature is ready.

To clarify, do you mean the goal is to creating checkpoint v5 even after the feature branch is merged to the master?  And another PR is needed to create checkpoint v6?  For Compactor or state extraction tool, or both?

If we just want to postpone generating checkpoint v6 (enabled by another PR), I think this PR should include `ReadCheckpointV6` because it doesn't affect writing checkpoints.